applications_tests/lethe-fluid/non-uniform_heat_flux_bc.prm

```diff
@@ -0,0 +1,131 @@
+  set temperature order = 2
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+
+subsection physical properties
+  set number of solids = 1
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+
+subsection mesh
+  set type               = dealii
+  set grid type          = hyper_rectangle
+  set grid arguments     = 0, 0 : 0.5, 1 : true
+  set initial refinement = 3
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+
+subsection multiphysics
+  set heat transfer       = true
+end
+
+#---------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+
+subsection mesh adaptation
+  set type = none
+end
+
+#---------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+
+subsection boundary conditions heat transfer
+  set number = 4
+  subsection bc 0
+    set id   = 0
+    set type = convection-radiation-flux
+    subsection heat_flux
+      set Function expression = 100*y
+    end
+  end
+  subsection bc 1
+    set id    = 1
+    set type  = temperature
+    subsection value
+      set Function expression = 80
+    end
+  end
+  subsection bc 2
+    set id    = 2
+    set type = convection-radiation-flux
+  end
+  subsection bc 3
+    set id    = 3
+    set type = convection-radiation-flux
+  end
+end
+
+subsection post-processing
+  set verbosity                        = verbose
+  set output frequency                 = 100000
+  set calculate temperature statistics = true
+  set temperature statistics name = non_uniform_heat_flux
+end
+
+#---------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+
+subsection non-linear solver
+  subsection heat transfer
+    set verbosity      = quiet
+    set tolerance      = 1e-1
+    set max iterations = 10
+  end
+  subsection fluid dynamics
+    set verbosity      = quiet
+  end
+end
+
+#---------------------------------------------------
+# Linear Solver Control
+#---------------------------------------------------
+
+subsection linear solver
+  subsection fluid dynamics
+    set verbosity                             = quiet
+  end
+  subsection heat transfer
+    set verbosity                             = verbose
+    set method                                = gmres
+    set max iters                             = 5000
+    set relative residual                     = 1e-7
+    set minimum residual                      = 1e-8
+    set preconditioner                        = ilu
+    set ilu preconditioner fill               = 0
+    set ilu preconditioner absolute tolerance = 1e-9
+    set ilu preconditioner relative tolerance = 1.00
```

Comment:
```suggestion

```
default parameters are ok here I think

Reply:
Maybe a good idea to check some other sections to keep it to the minimal, at least as best as you can